apis/cloud.redhat.com/v1alpha1/clowdenvironment_types.go

```diff
@@ -418,6 +418,7 @@ type MinioStatus struct {
 type ClowdEnvironmentStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+	Conditions      []ClowdCondition        `json:"conditions,omitempty"`
```

Comment:
I see that the root(ish) type has changed. We can change the `status` structs as we please, correct? We aren't tied to the same rigidity as the spec?

Reply:
This is an addition - and it's omit-empty